services/ui-src/src/forms/mcpar/mcpar.json

```diff
@@ -1458,7 +1458,20 @@
               "spreadsheet": "D1_Plan_Set"
             },
             "dashboardTitle": "Report program characteristics & enrollment for each plan",
-            "drawerTitle": "Report program characteristics & enrollment for"
+            "drawerTitle": "Report program characteristics & enrollment for",
+            "missingEntityMessage": [
+              {
+                "type": "span",
+                "content": "This program is missing plans. You won’t be able to complete this section until you’ve added all the plans that participate in this program in section A.7. "
+              },
+              {
+                "type": "internalLink",
+                "content": "Add Plans",
+                "props": {
+                  "to": "/mcpar/program-information/add-plans"
+                }
+              }
+            ]
```

Comment:
Note for reviewer: Same basic section added for each DrawerReportPage in the template.

Reply:
previously the content was in a shared variable, now it is repeated several times. Is there a reason to not have a const missingEntityMessage?

Reply:
yeah i considered doing that, but the reality is that we already have two types (plans (most of them) and bss entities (only 1 instance)) that are being used, and there will likely be others added in the future. because we have to do this in json, we'd have to set the plans-only verbiage somewhere in the form and then reference it, which wouldn't be all bad, but if we wanted to diverge from that on a single page we'd then be overwriting it, etc. it just seemed like more logic than it was worth to avoid repeated lines in a json file that's already 3k+. 🤷